### ISSUE TITLE
Washer not added to Smarthings

### ISSUE THREAD

**Issue by JoaoPedroBelo** [fison67/LG-Connector#16]
I have one washer in my lg connector but the washer is not showing up in the smartthings.

I Have the app and handlers installed (picures bellow).

Any ideas? 
![173679670_743260826293576_6422573623546275905_n](https://user-images.githubusercontent.com/30899202/115032182-f23e2300-9ec0-11eb-8d5b-10290c8e28b7.jpg)
![174062267_453433502578765_7232399266136209398_n](https://user-images.githubusercontent.com/30899202/115032184-f36f5000-9ec0-11eb-844d-ae1deca0bb17.jpg)

**Comment by JoaoPedroBelo** [comment]
fix it. Maybe update the docs about the ST Settings
Need to be updated with the info in smarttapp -> config

**Comment by albertovm** [comment]
> fix it. Maybe update the docs about the ST Settings
> Need to be updated with the info in smarttapp -> config

Hi João... could you give me a quick help? I have the same problem with my washer.

LG Connector appears to be OK, but I am not sure what to place in Smartthinq Token.
SmartApp appears and I did the configuration, like in your image.
I have my washer in https://www.lg.com/br/meu-suporte/meus-produtos.

I can't find the problem. Valeu!!

**Comment by JoaoPedroBelo** [comment]
you need to add the token in the connector page. you also need to add the config on the connector.
But to be honest this never worked very well. I moved to homeassistant.

**Comment by razor05** [comment]
> you need to add the token in the connector page. you also need to add the config on the connector.
> But to be honest this never worked very well. I moved to homeassistant.

when you say add the token in the connector page and the config on the connector, how did you do that?  

I went to the 'External Management' tab and it opens the connector - but i'm not sure what i do from there -- my devices show up on the dashboard, but not on the devices - and when i try to register them the only option is for a TV.

The 'Internal Management' tab doesn't open anything....did it for you?

Did your washer get added as a device then?

**Comment by bshaffer82** [comment]
Agreed, bite the bullet and move to home assistant.  I did it a year ago
with tons of customizations in smart things and it was the right decision
for me.

Craig
